Update lab3.md assignment text

milosonator opened the pull request:
Add an example on how layout in a syntax definition template is reflected in the pretty-printed output.

milosonator committed:
Update lab3.md assignment text 

Add an example on how layout in a syntax definition template is reflected in the pretty-printed output.

guwac commented:
An illustrating example would indeed be useful here. @milosonator or @eduardosamorim, can you add a MiniJava example instead, e.g. a class declaration? Should consist of an example class, simple template rule which results in ugly pretty-printing, and adjusted template rule which results in pretty pretty-printing.

milosonator committed:
Update lab3.md to include minjava example

Changed the sdf3 pretty-printing template example to MiniJava syntax.

milosonator committed:
Update lab3.md template example

Add a pre and post for pretty-printing template with and without nice layout, so the difference is clear.

milosonator commented:
I changed the example to a MiniJava example with normal template which results in poor pretty-printing and a template which makes the printed result more pretty.

milosonator commented:
@guwac @udesou If possible, please check out my changes.
